test/safe_pg_migrations_test.rb

```diff
@@ -411,7 +457,7 @@ def change
       # The foreign key is added.
       "SET statement_timeout TO '5s'",
       'ALTER TABLE "users" ADD CONSTRAINT "fk_rails_6d0b8b3c2f" FOREIGN KEY ("user_id") ' \
-        'REFERENCES "users" ("id")',
+        'REFERENCES "users" ("id") NOT VALID',
```

Comment:
@rchoquet could you validate this change please?

Reply:
Yes, this is what we want